htdocs/tools/textmessage.bml

```diff
@@ -39,7 +39,7 @@ body<=
     unless ($user) {
         $ret .= "<?h1 $ML{'.enter.user.head'} h1?>".
             "<?p " . BML::ml( '.enter.user.text3', { sitenameshort => $LJ::SITENAMESHORT } ) . " p?>".
-            "<div style='margin-left: 40px'><form method='get' action='textmessage'>$ML{'.enter.user.input'} <input type='text' size='15' maxlength='15' name='user' /> <input type='submit' value=\"$ML{'.enter.user.submit'}\" /></form></div>".
+            "<div style='margin-left: 40px'><form method='get' action='textmessage'>$ML{'.enter.user.input'} <input type='text' size='25' maxlength='25' name='user' /> <input type='submit' value=\"$ML{'.enter.user.submit'}\" /></form></div>".
             "<p>".BML::ml('.setup.text', { aopts => "href='$LJ::SITEROOT/manage/profile/'" })."</p>";
```

Comment:
If you decide not to add the variable here in this pass, please at least add a comment (similar to the one I used in cgi-bin/LJ/Session.pm) to help people remember to fix it when it gets converted to TT.